File: java/dxa-module-flex-layout/src/main/java/com/sdl/dxa/modules/flexlayout/model/CarouselContainer.java
package com.sdl.dxa.modules.flexlayout.model;

import com.sdl.webapp.common.api.mapping.semantic.annotations.SemanticEntity;
import com.sdl.webapp.common.api.mapping.semantic.annotations.SemanticProperty;
import com.sdl.webapp.common.exceptions.DxaException;

import static com.sdl.webapp.common.api.mapping.semantic.config.SemanticVocabulary.SDL_CORE;

/**
 * Carousel Container
 *
 * @author nic
 */
@SemanticEntity(entityName = "CarouselContainer",  vocabulary = SDL_CORE, prefix = "c")
public class CarouselContainer extends LayoutContainer {

    @SemanticProperty("c:interval")
    private int interval = 5000; // Default: 5000 ms

    public CarouselContainer() throws DxaException {
    }

    public int getInterval() {
        return interval;
    }
}

File: dotnet/SDL.DXA.Modules.FlexLayout/Models/CarouselContainer.cs
using Sdl.Web.Common.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SDL.DXA.Modules.FlexLayout.Models
{
    /// <summary>
    /// Carousel Container
    /// </summary>
    [SemanticEntity(EntityName = "CarouselContainer", Vocab = CoreVocabulary, Prefix = "c")]
    public class CarouselContainer : LayoutContainer
    {
        const int DEFAULT_INTERVAL = 5000; // Default 5000 ms 
        private int _interval = DEFAULT_INTERVAL;

        [SemanticProperty("c:interval")]
        public int Interval
        {
            get
            {
                return _interval;
            }
            set
            {
                _interval = value;
            }
        }
    }
}